docs/git_strategy.md

```diff
@@ -0,0 +1,28 @@
+# GIT guidelines
+
+## Regular branches
+
+### main
+- contains only code that works
+
+### feature branches
+- branched from **main**
+- naming convention `<branch type>-<brief description>-<task # opt>`
+- use dashes (`-`) between name parts and underscore (`_`) with name parts
+- branch type
+    - **feature** - feature branch with planned release
+    - **wip** - work in progress branch with no planned release
+- description - short description of changes in branch
+- task # - optional value used to tie back to a story
+
+*Example*: `feature-update_branch_strategy_docs-1`
+
+## Pull request
+The pull request should have a descriptive title. It should describe the changes, any furter action needed and if possible specify the testing.
```

Comment:
We have a little typo here: it should say "further".

Also, it's a matter of taste and style, but what do you think of using [semantic line breaks](https://sembr.org/)? In this case it would break this line in two lines:
```markdown
The pull request should have a descriptive title.
It should describe the changes, any further action needed and if possible specify the testing.
```